README.md

```diff
@@ -21,9 +21,9 @@ The Submitty project is hosted on [GitHub](https://github.com/Submitty).
 # Key Features
 
 * Secure testing of many languages: Python, C/C++, Java, Scheme, Prolog, SPIM, and anything available on GNU / Linux!
-* Customizable automated grading with immediate feedback to students
+* Customizable automated grading with immediate feedback to students.
```

Comment:
Periods are not necessary in this case as this is not a complete sentence. I believe part of the confusion comes from ! mark in the first item and etc. being an abbreviation for etcetera. Listed items do not necessarily need periods.